deploy/scripts/000100-revoke-roles.ts

```diff
@@ -0,0 +1,81 @@
+import {
+    grantRole,
+    InstanceName,
+    isLive,
+    renounceRole,
+    revokeRole,
+    setDeploymentMetadata
+} from '../../utils/Deploy';
+import { Roles } from '../../utils/Roles';
+import { DeployFunction } from 'hardhat-deploy/types';
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+
+const func: DeployFunction = async ({ getNamedAccounts }: HardhatRuntimeEnvironment) => {
+    const { deployer, daoMultisig } = await getNamedAccounts();
+
+    // Grant CarbonController admin roles to dao multisig
+    await grantRole({
+        name: InstanceName.CarbonController,
+        id: Roles.Upgradeable.ROLE_ADMIN,
+        member: daoMultisig,
+        from: deployer
+    });
+
+    await revokeRole({
```

Comment:
Nitpick: you can call `renounceRole` to be a bit more explicit

Reply:
I prefer `revokeRole` actually for the admin roles, makes for a bit more readable code.